lib/pinch_hitter/service/replay_ws.rb

```diff
@@ -86,12 +86,17 @@ def respond(endpoint='/', request=nil)
       endpoint = normalize(endpoint)
       body, request = wrap(request)
       @@recorder.record(endpoint, request)
-      message = @@handlers.respond_to(endpoint, body, request, response)
-      if message.is_a? String
-        content_type determine_content_type message
-        puts "No message found for #{endpoint}" unless message
+      if @@handlers.handler_for?(endpoint)
+        message = @@handlers.respond_to(endpoint, body, request, response)
+        if message
```

Comment:
Ooh, you uncovered a missing test ❤️   The type check was a cover for the `register_module` behavior - a user can write a module that  returns anything, but I didn't want to check content_type on things that would fail.

Reply:
Ah, okay. Easy enough to put back the type check around the `#content_type` call

Reply:
Looks like I was wrong here - this type check isn't needed anymore 👍 I'll merge as-is.
Follow on PR will have changelog entries.